src/lib/tooltip/tooltip-module.ts

```diff
@@ -22,5 +26,12 @@ import {MatTooltip, TooltipComponent} from './tooltip';
   exports: [MatTooltip, TooltipComponent, MatCommonModule],
   declarations: [MatTooltip, TooltipComponent],
   entryComponents: [TooltipComponent],
+  providers: [
+    {
+      provide: MAT_TOOLTIP_SCROLL_STRATEGY,
+      deps: [Overlay],
+      useFactory: MAT_TOOLTIP_SCROLL_STRATEGY_FACTORY,
+    }
```

Comment:
It looks like this object literal provider is also not working. We should just change all of them at this point.

Reply:
I should also mention that it's possibly an issue w/ Closure Compiler being too aggressive.